app/assets/javascripts/discourse/app/models/user.js

```diff
@@ -1171,33 +1171,42 @@ const User = RestModel.extend({
     return [...trackedTags, ...watchedTags, ...watchingFirstPostTags];
```

Comment:
This is not brought by this PR, but I would say this code shouldn't be in models/user